src/github.com/t3chguy/riot-static/workers.go

```diff
@@ -1,93 +0,0 @@
-// Copyright 2017 Michael Telatynski <7t3chguy@gmail.com>
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package main
-
-import (
-	//"fmt"
-	"github.com/t3chguy/riot-static/mxclient"
-	"hash/fnv"
-)
-
-type JobResp interface{}
-type Job interface {
-	Work(w *Worker)
-}
-
-type Worker struct {
-	ID     int
-	client *mxclient.Client
-	Queue  chan Job
-	Output chan JobResp
-	rooms  map[string]*mxclient.Room
-}
-
-func (w *Worker) Start() {
-	for {
-		job := <-w.Queue
-		//fmt.Println(job)
-		job.Work(w)
```

Comment:
as per conversation this'll need more thought as `select` applies to waiting on Chans, not methods